client/lib/api/user_preferences.dart

```diff
@@ -2,6 +2,9 @@ import 'dart:math';
 
 import 'package:firebase_analytics/firebase_analytics.dart';
 import 'package:firebase_messaging/firebase_messaging.dart';
+
+// import 'package:firebase_performance/firebase_performance.dart';
```

Comment:
Is this meant to be a TODO or just be removed?

Reply:
There is a TODO later in the file to uncomment the line to enable performance monitoring. When they uncomment it, it will throw an error because firebase performance will not be imported. this line is the dependency they need pre-commented so they can uncomment it and be done.